frontend/src/components/common/episode/episode.tsx

```diff
@@ -0,0 +1,50 @@
+import { useAppSelector, useDispatch, useLocation, useEffect } from 'hooks/hooks';
+import { RootState } from 'common/types/types';
+import { episodepage as episodeActions } from 'store/actions';
+import styles from './styles.module.scss';
+
+const Episode: React.FC = () => {
+  const dispatch = useDispatch();
+  const { pathname } = useLocation();
+  const episodeId = Number(pathname.split('/').pop());
+
+  useEffect(() => {
+    dispatch(episodeActions.loadEpisode(episodeId));
+  }, []);
+
+  const { episode } = useAppSelector(({ episodepage }: RootState) => ({
+    episode: episodepage.episode,
+  }));
+
+  const hasEpisode = Boolean(episode);
+
+  const episodeMock = {
+    podcastsGroup: 'English podcasts',
+    img: 'https://i.pinimg.com/474x/19/28/de/1928deaa3f4ba47822a5c1723f4b930f.jpg',
+    description: 'Aliens send Rick, Morty and Jerry into an alternate reality, and Rick tries to get them out as oblivious Jerry pitches a marketing slogan for apples.',
+  };
+
+  return (
+    <section className={styles.episode}>
+      <section className={styles.episodeHeader}>
+        {hasEpisode ? (
```

Comment:
Ideally, we should not check this here, this episode page should not appear if it's absent